hedera-node/configuration/mainnet/upgrade/feeSchedules.json

```diff
@@ -1925,6 +1925,52 @@
           ]
         }
       },
+      {
+        "transactionFeeSchedule": {
+          "hederaFunctionality": "TokenCancelAirdrop",
+          "fees": [
+            {
```

Comment:
Shouldn't this just be same price as `CryptoTransfer` `FUNGIBLE_COMMON` price?

Reply:
I think the fee is finalized 
```
TokenAirdrop (if the receiver does not have a prior association with the airdropped token): 0.10000
TokenClaimAirdrop: $0.00100
TokenCancelAirdrop: $0.00100
TokenReject: $0.00100
```

Reply:
Fixed 👍